packages/wonder-blocks-dropdown/src/components/multi-select.tsx

```diff
@@ -357,10 +357,12 @@ const MultiSelect = (props: Props) => {
             .filter((option) => !!option && !option.props.disabled)
             .map((option) => option.props.value);
         onChange(selected);
+        onSelectedValuesChangeValidation();
```

Comment:
Since the values change when the `Select none` / `Select all` options are picked, we call the `onSelectedValuesChangeValidation` handler